client/my-sites/domains/domain-management/privacy-protection/card/add-button.jsx

```diff
@@ -30,16 +31,18 @@ const AddButton = React.createClass( {
 				type="button"
 				className="button is-primary"
 				onClick={ this.addPrivacyProtection }>
-				{ this.translate( 'Add Privacy Protection' ) }
+				{ this.props.translate( 'Add Privacy Protection' ) }
 			</button>
 		);
-	},
+	}
 
-	addPrivacyProtection() {
-		upgradesActions.addItem( cartItems.domainPrivacyProtection( { domain: this.props.selectedDomainName } ) );
+	addPrivacyProtection = () => {
+		upgradesActions.addItem(
+			cartItems.domainPrivacyProtection( { domain: this.props.selectedDomainName } )
```

Comment:
`domain` which really means `selectedDomainName` or at least `domainName`.  It looks like every `domain` in `cart-items.js` really should be renamed. Probably for another PR.